readalongs/web_api.py

```diff
@@ -136,13 +136,9 @@ async def langs() -> List[SupportedLanguage]:
         ]
     """
     langs, lang_names = LANGS
-    return sorted(
-        (
-            SupportedLanguage(code=code, names=dict(_=lang_names[code]))
-            for code in langs
-        ),
-        key=lambda lang: lang.code,
-    )
+    return [
+        SupportedLanguage(code=code, names=dict(_=lang_names[code])) for code in langs
```

Comment:
I guess they are sorted elsewhere now?

Reply:
They have been sorted for quite a while, where the list is generated, i.e., in get_langs(), so it's a minor optimization to not sort again.

I've added a unit test to confirm we don't undo that at some point.

Reply:
And in fact, it Studio-Web, the first think we do with this list is sort it again by names instead of code, so it's silly to sort on twice on the wrong key before sending it over!